scrapper.py

```diff
@@ -26,16 +36,40 @@ class Crawler:
     Crawler implementation
     """
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.max_articles = max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        content = article_bs.find_all('div', id='main-content')
+        for article in content:
+            all_links = article.find_all('a')
+
+        for link in all_links:
+            try:
+                if link['href'][:4] == 'http':
+                    self.urls.append(link['href'])
+                else:
+                    self.urls.append('https://vja.ruslang.ru' + link['href'])
```

Comment:
it is better to use this method to solely extract urls from given soup
leave adding it to the attribute to find_articles
also, before adding a new url to the attribute it is necessary to check that such url has not been added before as many sources duplicate urls on their pages